server/mlabns/conf/tools.csv

```diff
@@ -4,5 +4,4 @@ ndt_ssl,iupui_ndt,,False
 mobiperf,michigan_1,,False
 npad,iupui_npad,8000,False
 ooni,mlab_ooni,,True
-glasnost,mpisws_broadband,19981,False
```

Comment:
what's the effect of changing this line vs removing glasnost from here: https://github.com/m-lab/mlab-ns/blob/master/server/mlabns/templates/map_view.html#L233 ?

Reply:
Shouldn't we also make the change on L233 ?

Reply:
Acknowledged. I will remove in my next commit.